### PR TITLE
Add coverage using bisect_ppx

### PR DESCRIPTION
**Pull request by punchagan**

### CODE REVIEW

File: pipeline/tests/dune [hunk 1]
```diff
@@ -4,7 +4,8 @@
    current_git current_github current_ocluster capnp-rpc-unix current_web
    dockerfile fmt.tty logs logs.fmt prometheus rresult uri yojson str)
  (preprocess
-  (pps ppx_deriving_yojson)))
+  (pps ppx_deriving_yojson))
+ (instrumentation (backend bisect_ppx)))
```
Comment (gs0510) [inline pipeline/tests/dune:8]:
```suggestion
 (instrumentation
  (backend bisect_ppx)))
```

**Review by gs0510 — APPROVED** [review]
LGTM, just one suggestion to make CI pass :)

File: pipeline/tests/dune (hunk 1 above)
Comment (punchagan) [reply pipeline/tests/dune]: Thanks, CI passes now. :)

**Comment by art-w** [comment]
Cool!

**Pull request merged**